airflow/models/xcom.py

```diff
@@ -124,85 +122,70 @@ def set(
 
     @classmethod
     @provide_session
-    def get_one(cls,
```

Comment:
Need a note in UPDATING about the removal of this method please

Reply:
Done 👍